docs/user/guides/hausanschluss-bundles.md

```diff
@@ -0,0 +1,240 @@
+
+### Bundle Overview
+
+The umbrella chart is organized into four main Capability Bundles (each a Helm subchart):
+
+| Bundle                 | Default Components                                      | Remappable via BYO                        |
+|------------------------|---------------------------------------------------------|-------------------------------------------|
+| Identity & Trust       | ssi-dim-wallet-stub                                     | Replaceable with your own Identity Wallet |
+| Dataspace Connector    | Eclipse Dataspace Connector, PostgreSQL, Vault subchart | Vault / PostgreSQL can be swapped out     |
+| Digital Twin           | Digital Twin Registry (DTR), PostgreSQL                 | PostgreSQL can be swapped out             |
+| Data Persistence Layer | Simple Submodel Server                                  | -                                         |
+
+You can enable or disable any bundle independently and supply your own implementations.
+
+## Installation Guide
+
+### Add the Helm Repository
+
+```bash
+helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
+helm repo update
+```
+
+### Update the Chart Dependencies
+
+```bash
+helm dependency update charts/data-persistence-layer-bundle
+helm dependency update charts/dataspace-connector-bundle
+helm dependency update charts/digital-twin-bundle
+helm dependency update charts/identity-and-trust-bundle
+```
+
+### Deploying the Charts
+
+#### Umbrella
+
+To install the entire umbrella Chart, see [README.md](/README.md).
+
+#### Install a single Bundle
+
+Install a bundle with its default values:
+
+##### dataspace-connector-bundle
+
+```bash
+helm install connector tractusx-dev/dataspace-connector-bundle \
+  --namespace umbrella --create-namespace
+```
+
+##### data-persistence-layer-bundle
+
+```bash
+helm install submodel-server tractusx-dev/data-persistence-layer-bundle \
+  --namespace umbrella --create-namespace
+```
+
+##### digital-twin-bundle
+
+> ⚠️ Due to an issue in the sldt-digital-twin-registry Helm Chart, the dataSource.url cannot be templated and has to be set manually to the release name
+
+```bash
+helm install dtr tractusx-dev/digital-twin-bundle \
+  --namespace umbrella --create-namespace \
+  --set digital-twin-registry.registry.dataSource.url=jdbc:postgresql://dtr-postgresql:5432/dtr
+```
+
+##### identity-and-trust-bundle
+
+```bash
+helm install wallet tractusx-dev/identity-and-trust-bundle \
+  --namespace umbrella --create-namespace
+```
+
+#### Use a bundle as dependencies
+
+Create your own Chart and add the bundles as dependency.
+
+```yaml
+dependencies:
+  - name: dataspace-connector-bundle
+    version: 1.0.1
+    repository: https://eclipse-tractusx.github.io/charts/dev
+```
+
+The Bundles are pre-configured to work with in the Umbrella environment.  
+In case you want to use them in your own dataspace, you have to adjust the "Identity and Trust Protocol (IATP) Settings"
```

Comment:
```suggestion
In case you want to use them in your own dataspace, you have to adjust the Decentralized Claims Protocol (formerly IATP) Settings"
```